### ISSUE TITLE
[Feature request] Locally save post links

### ISSUE THREAD

**Issue by Latrolage** [zedeus/nitter#570]
Option to save links to post in local storage or cookies so you can go back to those links later.

**Comment by zedeus** [comment]
Why not just create bookmarks?

**Comment by Latrolage** [comment]
That's true, I was just thinking that a scrollable list of posts would look better.

After rereading my initial post again: Sorry, my initial post made it seem like i wanted a list of literally just the link to posts. I meant a page which had saved posts/comments which you could scroll through without having to open new tabs etc.

**Comment by zedeus** [comment]
Local storage requires JavaScript and I don't think cookies are a good way to store a potentially very long list of IDs. Having a list of "favorited/liked" tweets that can be accessed is something I've wanted to do, but it would probably require a real account system, and I'd like to archive the favorited tweets as well.
